### DISCUSSION TITLE
Restraining protein-ligand distance (residue-wise or atom-wise)

### DISCUSSION THREAD

**Discussion by YoavShamir5** [chaidiscovery/chai-lab#177] [Q&A] [answered]
Hi, Incredible work with Chai-1! I have two related queries:
1) Can one restraint (pocket/contact) the distance between a residue and a small-molecule ligand? If so - what should be used as "res_idxB" for the ligand? 
2) Is there an example of the syntax to restraint distance between atom A of the ligand and atom B of the protein? I've read some discussions on this here, but am not sure what's the relevant syntax for a .csv input. 
Thanks!

**Comment by arogozhnikov** [comment, marked as answer]
From syntax perspective, format works like this:
res_idx for atom: `K123@CB`
res_idx for ligand: `@C2`

that said, model was not trained to deal with atom-level restraints, so this syntax is *future-looking*

And it's all here https://github.com/chaidiscovery/chai-lab/blob/main/chai_lab/data/parsing/restraints.py
